### DISCUSSION TITLE
Patterns for connecting multiple servers to logdy?

### DISCUSSION THREAD

**Discussion by ross-spencer** [logdyhq/logdy-core#24] [Q&A]
Is there a way, or is anyone using Logdy as a central logging utility for multiple servers? Is it supported by the app, or are folks using other approaches to make this happen?

**Comment by suntong** [comment]
+1, came here for the same reason.

From [doc](https://logdy.dev/docs/reference/cli#ip-used-with-forward-and-socket-mode-only),

> IP (used with forward and socket mode only)
> --ip=xxxx - IP address to listen to, leave empty to listen on all IP addresses.

The _"all IP addresses"_ in above doc means all IP interfaces the network card is on, right, or it truly means that  logdy is capable of listening to (local) networks?

**Reply by PeterOsinski** [reply]
@suntong it means that Logdy will bind to all local network interfaces. For example, if you machine is reachable through multiple IP addresses, then by typing these addresses in the browser along with the port will load the UI.

**Comment by PeterOsinski** [comment]
Hi @ross-spencer currently you have to coordinate fetching the logs to a central place by yourself. I'll make a note for myself on this and see if the community would be open to welcome this feature

**Reply by lwmacct** [reply]
in great need of
